portfolio/src/main/webapp/script.js

```diff
@@ -12,14 +12,27 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
+/** Retrieves data from the /data page and displays it. */
+function getComments() {
+  fetch('/data').then(response => response.json()).then((json) => {
+    json.forEach(commentText => createPElement(commentText));
+  });
+}
+
+/** Creates paragraph elements and sets the text to comments 
+ *   pulled from \data page. 
  */
+function createPElement(commentText) {
+  const commentElement = document.createElement("p");
+  commentElement.innerText = commentText
+  document.body.appendChild(commentElement);
+}
 
-// Cycles through 3 different pictures of myself.
 let pictureNumber = 0;
+
+/** Cycles through 3 different pictures of myself. */
 function cyclePictures() {
   const headerSelfie = document.getElementById("header-selfie");
   const imageNames = ["campus", "suit", "main-selfie"];
   headerSelfie.src = `images/${imageNames[pictureNumber++ % imageNames.length]}.jpg`;
-}
+}
```

Comment:
Nit: have 1 empty line at the end of the file

Reply:
Done.